content/docs/07-reference/ai-sdk-core/02-stream-text.mdx

```diff
@@ -590,6 +597,55 @@ for await (const textPart of textStream) {
             },
           ],
         },
+        {
+          type: 'TextStreamPart',
+          parameters: [
+            {
+              name: 'type',
+              type: "'tool-call-streaming-start'",
```

Comment:
suggestion: could be `tool-call-start`, `streaming` in the name sounds redundant

Reply:
was thinking about it, would it be clear that `start` refers to the start of the streaming (vs. the potential tool execution)?

Reply:
some context about trade-offs:

![CleanShot 2024-07-18 at 10 07 19](https://github.com/user-attachments/assets/5072a74c-1c47-4d40-86d7-9d5b068ee91b)

i'm leaning towards the longer option for clarity, since it's not an api most users will use. i expect that using fullStream is the exception

Reply:
ah, makes sense to have it called `tool-call-streaming-start` for clarity